assignments/prototypes.js

```diff
@@ -15,6 +15,7 @@
   * dimensions (These represent the character's size in the video game)
   * destroy() // prototype method that returns: `${this.name} was removed from the game.`
 */
+function GameObject()
```

Comment:
make sure you actually create the prototype and object, like this
function GameObject(options) {
  this.createdAt = options.createdAt;
  this.dimensions = options.dimensions;
  this.name = options.name;
}

GameObject.prototype.destroy = function () {
  return `${this.name} was removed from the game.`;
};

/*
=== CharacterStats ===
* healthPoints
* name
* takeDamage() // prototype method -> returns the string '<object name> took damage.'
* should inherit destroy() from GameObject's prototype
*/

function CharacterStats(characterStatsOptions) {
  GameObject.call(this, characterStatsOptions);
  this.healthPoints = characterStatsOptions.healthPoints;
}

// Sets up inheritance with GameObject
CharacterStats.prototype = Object.create(GameObject.prototype);

CharacterStats.prototype.takeDamage = function () {
  return `${this.name} took damage.`;
};

Reply:
function Humanoid(humanoidOptions) {
  CharacterStats.call(this, humanoidOptions);
  this.team = humanoidOptions.team;
  this.weapons = humanoidOptions.weapons;
  this.language = humanoidOptions.language;
}

Humanoid.prototype = Object.create(CharacterStats.prototype);

Humanoid.prototype.greet = function () {
  return `${this.name} offers a greeting in ${this.language}.`;
};